docs/prem-app/network-mode.md

```diff
@@ -0,0 +1,46 @@
+---
+id: network-mode
+title: Contributing to the Prem Network
+sidebar_label: Network Mode
+sidebar_position: 3
+---
+
+# Network Mode
+
+If you have installed Prem App on your Mac (Intel and Apple Silicon), you can contribute to the Prem Network by running a Prem node on your machine. In order to do that, you can follow the instructions below.
+
+## 1. Install Prem on your Mac
+
+You can check how to install Prem App on your mac [here](/docs/prem-app/installation/install-macos).
+
+## 2. Join the Network
+
+Go to `Settings` and scroll down to `Prem Network` section.
+
+![Network Init](./assets/network-init.png)
+
+- Select the Model you wanna contribute to. 
+- Select the number of blocks you want to contribute to.
+- Click on `Start` button.
+
+As soon as you Click on the `Start` button, the Prem App will start downloading the blocks from the network and will start contributing to the network. The process requires a coouple of minutes to complete. In the meantime, you will see the following message:
+
+![Network Install](./assets/network-install.png)
+
+When the process is completed, a pop-up will appear with the following message:
```

Comment:
actually when the pop-up appears it still needs to install requirements the first time (see [this](https://github.com/premAI-io/prem-app/pull/440#discussion_r1374955214)

Reply:
Should we specify this? I am not sure.

Reply:
Yes maybe it's too much and not necessary